<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionGroupBox</class>
 <widget class="QGroupBox" name="PrescriptionGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string notr="true">Prescription Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="amountTitle">
       <property name="toolTip">
        <string notr="true">Amount</string>
       </property>
       <property name="text">
        <string notr="true">Amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="amountLabel">
       <property name="toolTip">
        <string notr="true">Amount</string>
       </property>
       <property name="text">
        <string notr="true">PH AMOUNT</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="writtenTitle">
       <property name="toolTip">
        <string notr="true">Written</string>
       </property>
       <property name="text">
        <string notr="true">Written:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="writtenLabel">
       <property name="toolTip">
        <string notr="true">Written</string>
       </property>
       <property name="text">
        <string notr="true">PH WRITTEN</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="filledTitle">
       <property name="toolTip">
        <string notr="true">Filled</string>
       </property>
       <property name="text">
        <string notr="true">Filled:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="filledLabel">
       <property name="toolTip">
        <string notr="true">Filled</string>
       </property>
       <property name="text">
        <string notr="true">PH FILLED</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="instructionsTitle">
       <property name="toolTip">
        <string notr="true">Prescription instructions</string>
       </property>
       <property name="text">
        <string notr="true">Instructions:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="instructionsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string notr="true">Prescription instructions</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string notr="true">PH INSTRUCTIONS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="validTitle">
       <property name="toolTip">
        <string notr="true">Active or inactive status</string>
       </property>
       <property name="text">
        <string notr="true">Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="validLabel">
       <property name="toolTip">
        <string notr="true">Active or inactive status</string>
       </property>
       <property name="text">
        <string notr="true">PH VALID</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
